README.md

```diff
@@ -54,5 +54,18 @@ var fan = smc.getFan(0);
 // => { name: 'ExhaustZ', speed: 2176 }
 ```
 
+### `smc.getTemp(sensor, units)`
+Gets the temperature value related to a specific sensor. `sensor` is a valid SMC
+sensor name (ex: `'TC0P'`). `units` is the string `'celsius'`, `'fahrenheit'`, or
```

Comment:
I'm not too fond of the unit selection, but I'm not sure if there is a better way here... Do you have any thoughts? I feel like adding case-insensitive full-string arguments are unnecessary, i.e. just `'F'` would be fine for Fahrenheit, although it's not a big step forward. It looks most like [SMC](https://github.com/beltex/libsmc/blob/ba8e75a8b2ad70d195090f98a4e426d35bd16616/src/smc.c#L542), yes, but it doesn't look like the JS port. What do you think?

Reply:
I honestly wasn't either. Single letter would work, it's certainly easier to type. I, personally, still prefer to keep it case-insensitive, because it's nice to not have to think about that while using it.

For now, I'll change to single character.